spec/action_spec.rb

```diff
@@ -104,6 +104,24 @@
     it_behaves_like 'version_increased? for all supported file types', '1.2.4', true
     it_behaves_like 'version_increased? for all supported file types', '1.3.0', true
     it_behaves_like 'version_increased? for all supported file types', '2.0.0', true
+
+    context 'when version file name has changed so old version file not found' do
+      it 'returns true if old version file not found' do
```

Comment:
You already mentioned `old version file not found` in context so can remove it and just `return true` here.